### PR TITLE
Remove fiam scope tokens

### PR DESCRIPTION
**Pull request by eldhosembabu**
Remove fiam scoped tokens from keychain set by old FIAM SDKs.

internal bug : b/284207019

### CODE REVIEW

File: FirebaseMessaging/Sources/FIRMessagingConstants.h [hunk 1]
```diff
@@ -59,6 +59,8 @@ FOUNDATION_EXPORT NSString *const kFIRMessaging_CMD_RST;
 FOUNDATION_EXPORT NSString *const kFIRMessagingAllScopeIdentifier;
 /// The scope used to save the IID "*" scope token.
 FOUNDATION_EXPORT NSString *const kFIRMessagingDefaultTokenScope;
+/// Blacklisted "fiam" token scope.
```
Comment (paulb777) [inline FirebaseMessaging/Sources/FIRMessagingConstants.h:62]: Use "Denylist" instead - here and throughout.

Comment (eldhosembabu) [reply FirebaseMessaging/Sources/FIRMessagingConstants.h]: Updated comments.

File: FirebaseMessaging/Sources/Token/FIRMessagingTokenInfo.m [hunk 2]
```diff
@@ -81,6 +81,10 @@ - (BOOL)isFreshWithIID:(NSString *)IID {
     return NO;
   }
 
+  if ([self hasBlacklistedScope]) {
```
Comment (paulb777) [inline FirebaseMessaging/Sources/Token/FIRMessagingTokenInfo.m:84]: Methods should be updated as well.

Comment (eldhosembabu) [reply FirebaseMessaging/Sources/Token/FIRMessagingTokenInfo.m]: Done

**Review by paulb777 — APPROVED** [review]

File: FirebaseMessaging/CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,6 @@
+# 10.11.0
```
Comment (paulb777) [inline FirebaseMessaging/CHANGELOG.md:1]: 10.12.0

Comment (eldhosembabu) [reply FirebaseMessaging/CHANGELOG.md]: Done

**Pull request merged**